docs/Bioinformatic-Analyses/RNAseq-on-Cavatica/rna_seq_9.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: page
+title: Incorporating GTEx Data in Kids First Analyses
+---
+
+<div class="banner"><span class="banner-text">Lesson in Development</span></div>
+
+Incorporating GTEx Data in Kids First Analyses
+==========================
+
+The Kids First DRC is a member of the [NIH Cloud Platform Interoperability Effort (NCPI)](https://anvilproject.org/ncpi), a collaboration of four NIH-funded DCCs to establish and implement guidelines and technical standards to empower end-user analyses across participating cloud platforms and facilitate the realization of a trans-NIH, federated data ecosystem. In brief, the goal of the NCPI Effort is to allow any authenticated user access to all of the platforms' datasets and the ability to work in the user's analysis platform of choice.
+
+As part of the NCPI Effort, methods have been developed to allow users to bring datasets from the other NIH platforms into CAVATICA for combined research projects. One external dataset that is supported is the [Genotype Tissue Expression (GTEx) Program](https://commonfund.nih.gov/gtex). GTEx was funded to study the relationship between genetic variants (inherited changes in DNA sequence) and gene expression (how genes are turned on and off) in multiple human tissues and across individuals. Their datasets can serve as great controls for RNA-Seq experiments, comparing expression in GTEx's "normal" brain tissue to Kids First's brain cancer tissue.
+
+The previous example provides a walkthrough for identifying RNA-Seq datasets from the [Kids First Data Resource Portal](https://portal.kidsfirstdrc.org/) and pushing them to [CAVATICA](https://cavatica.sbgenomics.com/) for analysis. This supplemental lesson will demonstrate how to find GTEx datasets on the [NHGRI Analysis Visualization and Informatics Lab-space (AnVIL)](https://anvilproject.org/) Portal] and import these files into a CAVATICA project for a combined analysis with Kids First data.
+
+
+## Step 1: Identify GTEx Files on AnVIL and Export a PFB
+- Navigate to [the AnVIL Portal](https://gen3.theanvil.io/login) and login using NIH or Google credentials.
+    - <img src="./rna-seq-images/rna-seq-9-01.png" width="400"/>
+- Using the [AnVIL Data Explorer](https://gen3.theanvil.io/explorer), identify the GTEx files for your analysis. For more information on using AnVIL Gen3, see [their support documentation here](https://anvilproject.org/learn/introduction/getting-started-with-gen3).
+    - <img src="./rna-seq-images/rna-seq-9-02.png" width="400"/>
+- Once you have identified the subset of files for your analysis, return to the top of the screen and choose `Export All to PFB`. This will create a manifest of files on the AnVIL Portal. This process may take several minutes, depending upon the number of files for export.
```

Comment:
On my screen, the button text is `Export to PFB` (without the "All")

![Screen Shot 2022-01-28 at 9 09 58 AM](https://user-images.githubusercontent.com/7544197/151591155-ea23b085-435a-42b0-b8e0-7afc041f256f.png)

Reply:
line 23 changed `Export All to PFB` to `Export to PFB`